packages/web-components/fast-ssr/README.md

```diff
@@ -1,7 +1,150 @@
 # FAST SSR
-This package contains tools to render FAST components outside the browser. More details to follow...
 
-## Testing
-This package uses Playwright and a lightweight web server for running tests. You can run the tests by running `npm run test`.
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![npm version](https://badge.fury.io/js/%40microsoft%2Ffast-ssr.svg)](https://badge.fury.io/js/%40microsoft%2Ffast-ssr)
 
-> Playwright may prompt you to install browsers to run the tests. If so, follow the instructions provided or run `npm run install-playwright-browsers`.
\ No newline at end of file
+The `@microsoft/fast-ssr` package contains a NodeJS solution for rendering FAST templates and components. While primarily intended for supporting SSR scenarios, it also allows FAST to be used as a general purpose HTML templating solution.
```

Comment:
I think we should spell this out at least once.

```suggestion
The `@microsoft/fast-ssr` package contains a NodeJS solution for rendering FAST templates and components. While primarily intended for supporting server-side rendering (SSR) scenarios, it also allows FAST to be used as a general purpose HTML templating solution.
```

Reply:
Yep, good call